e2e_tests/e2e_test_setup.sh

```diff
@@ -75,3 +75,14 @@ echo "Bitcoin Asset ID: "$btc_asset_id
             --elementsd http://admin1:123@localhost:18884 \
             --usdt $usdt_asset_id > e2e_tests/bobtimus.log 2>&1 &
 )
+
+# give bobtimus time to start:
+
+sleep 5s
+
+# fund test wallets
+curl -POST  http://localhost:3030/api/faucet/el1qqfc6m8nrnrfpcsj9thg2df9jzpv4pwm4shm5wvn2untay2dmhhk4ut8j35jtd7jvqplx3gt9zth7ndtn6mcq68d03l4e7wqay -d {}
```

Comment:
it might help to add a comment here to know which test uses which wallet and which one is bobtimus (I assume the first one, because that address is not used in the wallet tests...)

Reply:
Well, I guess that's not needed anymore :D